client/app/scripts/liveblog-bloglist/module.js

```diff
@@ -12,6 +12,7 @@
         ];
         $scope.activeState = isArchivedFilterSelected ? $scope.states[1] : $scope.states[0];
         $scope.creationStep = 'Details';
+        $scope.requestAccessMessage = gettext('Click to request access');
```

Comment:
haha, thank you for having tried :)

On Fri, Oct 16, 2015 at 1:42 PM Vlad Nicoara <notifications@github.com>
wrote:

> In client/app/scripts/liveblog-bloglist/module.js
> <https://github.com/superdesk/liveblog/pull/246#discussion_r42233585>:
>
> > @@ -12,6 +12,7 @@
> >          ];
> >          $scope.activeState = isArchivedFilterSelected ? $scope.states[1] : $scope.states[0];
> >          $scope.creationStep = 'Details';
> > +        $scope.requestAccessMessage = gettext('Click to request access');
>
> nope, not working... just tried it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/superdesk/liveblog/pull/246/files#r42233585>.
>